x-pack/plugins/ml/public/application/settings/filter_lists/list/filter_lists.js

```diff
@@ -85,6 +88,7 @@ export class FilterListsUI extends Component {
   render() {
     const { filterLists, selectedFilterLists } = this.state;
     const { canCreateFilter, canDeleteFilter } = this.props;
+    const helpLink = getDocLinks().links.ml.guide;
```

Comment:
Is it worth linking to the Custom rules [page](https://www.elastic.co/guide/en/machine-learning/master/ml-rules.html) here rather than the top level of the ML guide?